javascript/javascript1/week2/homework-js1-w2.js

```diff
@@ -0,0 +1,168 @@
+// freeCodeCamp username: yuejiaoshi
+// freeCodeCamp profile url: https://www.freecodecamp.org/yuejiaoshi
+
+//*****************************************/
+// Flight booking fullname function
+//*****************************************/
+function getFullname(firstname, surname, useFormalName = false, gender) {
+  if (!useFormalName) {
+    return firstname + " " + surname;
+  } else {
+    if (gender === "man") {
+      return "Lord" + " " + firstname + " " + surname;
```

Comment:
Suggestion in case you implement the `fullName` variable for reducing repetition.
```suggestion
      return `Lord ${fullName}`;
```